fe/src/pages/profile.vue

```diff
@@ -16,52 +16,23 @@
       />
     </div>
     <div v-if="activeTab === 'My Campaigns'">
-      <div
-        class="mt-6 grid grid-cols-3 gap-6 sm:grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 campaigns"
-      >
-        <CampaignCard
-          v-for="i in 6"
-          :key="i"
-          :title="title"
-          :description="description"
-          :eth-value="ethValue"
-          :img-src="imgSrc"
-          :days-left="daysLeft"
-        ></CampaignCard>
-      </div>
-      <div class="w-full mt-8 h-14 flex items-center justify-center">
-        <BasePaginator
-          :current-page="currentPage"
-          :last-page="10"
-          :per-page="first"
-          :on-page-change="setPage"
-        />
-      </div>
+      <UserCampaigns />
+    </div>
+    <div v-if="activeTab === 'Donators'">
+      <UserDonators />
```

Comment:
If each tabs contents are separated in components we can do like this
```suggestion
    <component :is="activeTab " />
```

then in the script 
```

import CompA from './CompA.vue'
import CompB from './CompB.vue'
import { ref, markRaw } from 'vue'

function changeTab (comp) {
  activeTab .value = markRaw(comp)
}

const profileTabs = [{ compoenent: CompA , label:  "My Campaigns"}, compoenent: CompB, label:  "Donations"})"];
const activeTab = ref(markRaw(CompA ));
```